### PR TITLE
feat(grid): embed bootstrap 4 grid system - INNO-314

### PR DESCRIPTION
**Pull request by yhuard**
This is a grid system based on BS 4 grid.

TODO:

- [x] Provide examples / docs / guidelines
- [ ] Convert breakpoints into rem units?

### CODE REVIEW

**Comment by yhuard** [comment]
If we use rem units, then we won't be able to write media queries with `(breakpoint)_max` because `(breakpoint)_max = (next breakpoint)_min - 1px`. But is it a real issue? When you write media queries, you generally go from small to big, so when 2 conditions overlap the last wins.

File: framework/base/_mixins.scss [hunk 1]
```diff
@@ -1,287 +1,9 @@
...
-
-  @else if $point == screen-xs-min {
-    @media (min-width: $screen-xs-min) {
-      @content;
-    }
-  }
-}
-
-// Select list arrow styling.
-@mixin select-list-arrow($background-color, $arrow-grey:false) {
```
Comment (kalinchernev) [inline framework/base/_mixins.scss]: Didn't find this migrated, is there a reason to skip it?

Comment (yhuard) [inline framework/base/_mixins.scss]: It is! https://github.com/ec-europa/europa-component-library/pull/89/files#diff-bb6c6a55e1b5000a357d826fdc115276R11

Comment (kalinchernev) [inline framework/base/_mixins.scss]: Alright, my search was scoped only to the mixins folder, but that's even better!

File: framework/base/_variables.scss [hunk 2]
```diff
@@ -1,21 +1,10 @@
-@import 'variables/breakpoints';
-@import 'variables/colors';
-@import 'variables/font-scale';
-@import 'variables/spacing';
+// Import all ECL base variables
 
-$font-family-sans-serif: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Fira Sans', 'Droid Sans', 'Helvetica Neue', Arial, sans-serif !default;
```
Comment (kalinchernev) [inline framework/base/_variables.scss]: Good job on splitting these nogs into more manageable small portions!

**Comment by kalinchernev** [comment]
@yhuard very basic documentation included, relying mostly on existing one of bootstrap.
Let's make developers happy again!

**Comment by yhuard** [comment]
Thanks, I'll review your changes and update the refs to make travis green again

**Comment by yhuard** [comment]
What about converting the breakpoints to REM units? @kalinchernev

**Comment by kalinchernev** [comment]
@yhuard rems are not a deal breaker for the moment (for me)

**Comment by yhuard** [comment]
Ok let's say we can merge it

**Pull request merged**